components/Checkout/VsfPaymentProvider.vue

```diff
@@ -3,37 +3,47 @@
     <SfHeading
       data-cy="svsf-vsfPaymentProvider-heading"
       :level="3"
-      :title="$t('Payment methods')"
+      :title="$t('Payment')"
       class="sf-heading--left sf-heading--no-underline title"
     />
-
-    <SfRadio
-      :data-cy="`svsf-vsfPaymentProvider-paymentMethods-radio-${index}`"
-      v-for="(method, index) in paymentMethods.list"
-      :key="index"
-      :label="paymentProviderGetters.getMethodName(method)"
-      :value="paymentProviderGetters.getMethodName(method)"
-      :description="paymentProviderGetters.getMethodDescription(method)"
-      :selected="paymentProviderGetters.getSelected(paymentMethods)"
-      name="paymentMethod"
-      class="form__radio payment"
-      @input="selectMethod(method)"
-    >
-      <div class="payment__label">
-        {{ paymentProviderGetters.getMethodName(method) }}
+    <template v-for="(method, index) in paymentMethods.list">
+      <div :key="index" class="sf-radio__wrapper" v-bind:class="{ 'is-active': paymentProviderGetters.getSelected(paymentMethods) === paymentProviderGetters.getMethodName(method) }">
```

Comment:
using index as key is not a good practice, meybe in `method` object is better property to be used as key?